pkg/apis/operator/v1alpha1/tektonresult_lifecycle.go

```diff
@@ -85,6 +85,10 @@ func (trs *TektonResultStatus) MarkInstallerSetNotReady(msg string) {
 		"Installer set not ready: %s", msg)
 }
 
+func (trs *TektonResultStatus) MarkPostReconcilerComplete() {
+	resultsCondSet.Manage(trs).MarkTrue(PostReconciler)
```

Comment:
You have to add it in conditionset at top also

Reply:
Done